### ISSUE TITLE
very low hashrate

### ISSUE THREAD

**Issue by erkange** [genesismining/sgminer-gm#114]
I use below settings but get only 20 mh/s per each amd rx570 cards.
What should be the correct settings?

ps: I get 29.8 wity claymore

my settings are like below

`{
	"pools":
	[
		{
			"name": "Ethermine",
			"url": "stratum+tcp://eu1.ethermine.org:4444",
			"user": "MYWALLETADDRESSISHERE.amdsgminer",
			"pass": "x",
			"profile": "eth"
		}
	],
	"profiles":
	[
		{
			"name": "eth",
			"algorithm": "ethash",
			"xintensity": "1024",
			"worksize": "192",
			"gpu-threads": "1"
		}
	],

"no-extranonce": "true",
"default-profile": "eth",
"api-listen": "true",
"temp-target": "72",
"gpu-fan": "45-85",
"gpu-engine" : "1150",
"gpu-memclock" : "2050",
"api-port": 4028,
"temp-cutoff": "82",
"api-listen": true,
"auto-fan": true,
"temp-overheat": "85",
"gpu-platform": "1"
}`

**Comment by 3cola** [comment]
Hello,

I have some rx470 and I do 24 mh/s with each cards.

I flashed the bios of my cards and did the common memory stripes (copy 1500 Mhz value to all above until 2000 Mhz)

those are my profile settings :

 "profiles": [
    {
      "name": "eth",
      "algorithm": "ethash-new",
      "device": "all",
      "shaders":"2048",
      "xintensity": "512,512,512,512,512",
      "worksize": "128",
      "gpu-engine": "1020,990,1019,990,1020",
      "gpu-memclock": "1725,1700,1725,1700,1725",
      "gpu-fan": "20-90"
    }

settings really depends on your cards, so not easy to tell...

you should try to follow the guide "tuning" in the docs folder.

the most important is to get the last AMD drivers (17.5). They include real optimizations.

**Comment by erkange** [comment]
24 is also very low. Claymore can give 29-30 MH/s

**Comment by 3cola** [comment]
yeah... It depends of your card, memory, etc... I have unfortunately bad Micron memory chips, I cannot overclocked them that much without everything getting unstable.
I am just telling you that for my current setup 5x RX 470 , Claymore is delivering 116 MH/s while consumming 900W and sgminer is delivering 122 MH/s while consumming 700W.
Maybe I messed up the Claymore setup...
